pkg/daemon/update.go

```diff
@@ -369,6 +400,7 @@ func Reconcilable(oldConfig, newConfig *mcfgv1.MachineConfig) (*MachineConfigDif
 	return &MachineConfigDiff{
 		osUpdate: oldConfig.Spec.OSImageURL != newConfig.Spec.OSImageURL,
 		kargs:    !reflect.DeepEqual(oldConfig.Spec.KernelArguments, newConfig.Spec.KernelArguments),
+		fips:     oldConfig.Spec.Fips != newConfig.Spec.Fips,
```

Comment:
This will force the reboot so that the change takes effect, correct?

Reply:
nope, the sync itself does the reboot the node (and it can't be avoided today as we know). This is just to log what we're changing during a sync.

Reply:
WFM. Just wanted to ensure a reboot does happen when FIPS changes :+1:

Reply:
👍 the e2e added here also takes care of validating fips has been enabled on nodes so it will fail if we skip a reboot after enabling fips (iiuic)